modules/oe_webtools_page_feedback/tests/src/Functional/PageFeedbackFormTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_page_feedback\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the configuration form of the Page Feedback Form webtools widget.
+ */
+class PageFeedbackFormTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_webtools_page_feedback',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Tests Page Feedback Form configuration.
+   */
+  public function testPageFeedbackForm(): void {
+    $user = $this->createUser([
+      'administer webtools page feedback form',
+    ]);
+    $this->drupalLogin($user);
+    $this->drupalGet('/admin/config/system/oe_webtools_page_feedback');
+    // Assert default values.
+    $this->assertSession()->pageTextContains('Webtools Page Feedback Form settings');
+    $this->assertSession()->checkboxNotChecked('Enabled');
+    $this->assertSession()->fieldValueEquals('Form ID', '');
+    // Configure the form.
+    $page = $this->getSession()->getPage();
+    $page->checkField('Enabled');
+    $page->fillField('Form ID', '1234');
+    $page->pressButton('Save configuration');
+    // Assert values are correctly saved.
```

Comment:
Lets just add a quick assertion to check that the form shows the stored values as well.